lib/features/composer/presentation/view/mobile/mobile_editor_view.dart

```diff
@@ -27,6 +28,7 @@ class MobileEditorView extends StatelessWidget with EditorViewMixin {
     this.arguments,
     this.contentViewState,
     this.onEditorContentHeightChanged,
+    this.onTextSelectionChanged,
```

Comment:
I would LOVE to have a separated commit for adding `onTextSelectionChanged` to the composer.